c2corg_ui/static/js/editing/articleediting.js

```diff
@@ -46,11 +46,53 @@ goog.inherits(app.ArticleEditingController, app.DocumentEditingController);
  * @param {appx.Document} data
  * @return {appx.Document}
  * @override
- * @public
```

Comment:
Why are removing the `public` annotation?

Reply:
This was a mistake, corrected by the last version.